docs/source/native/class.rst

```diff
@@ -0,0 +1,87 @@
+Classes
+=======
+
+
+
+.. note::
+
+   This is by no means complete. Please add anything you know.
+
+Declaring Classes
+-----------------
+
+To declare a class, first add the ``untyped`` keyword and the class as a global variable at file level.
+
+.. code-block:: javascript
+
+    untyped
+    global var ExampleClass
+
+You can then add the declaration inside of a function.
+
+Most classes use a constructor. A constructor is a function of the instance that gets executed on object creation.
+
+.. code-block:: javascript
+
+    void function initClient() {
+        class ExampleClass {
+            constructor(){print("Instance of ExampleClass created");}
+        }
+    }
+
+You can require parameters in the constructor. Keep in mind that you have to pass those when creating an object.
```

Comment:
Would be nice to include an example of how to this this, and how to use those variables later on